include/librealsense2/h/rs_pipeline.h

```diff
@@ -41,6 +41,15 @@ extern "C" {
     */
     void rs2_pipeline_stop(rs2_pipeline* pipe, rs2_error ** error);
 
+    /**
+     * Set the device to be used in the pipline.
+     * The function is used to assign the device, ensuring the pipeline maintains the sensitivity values configured for
+     * this device by the user. 
+     * \param[in] pipe the pipeline
+     * \param[in] device  the device to be used in the pipline.
+     */
+    void rs2_pipeline_set_device( rs2_pipeline * pipe, rs2_device * device, rs2_error ** error );
```

Comment:
Can we access it from python?

Reply:
Please add a unit test and verify on the way :)

Reply:
Added unit test